witms/spiders/guardian.py

```diff
@@ -1,24 +1,30 @@
 from scrapy import Request, Spider
-from scrapy.loader import ItemLoader
-
 from witms.items import Article
+from witms.loaders import ArticleLoader
 
 
 class GuardianSpider(Spider):
-    name = "guardian"
+    name = "Guardian"
```

Comment:
I really don't see the point in title casing this. This is the spider's name with which we can call a crawling job from the CLI by doing `scrapy crawl spider-name`, and I'm really not a fan of case-sensitive arguments...

Reply:
I was trying to reuse the bot's name as the Portal's name.

Other options would include:
1. Using str.title() for this bot and str.upper() for the BBC one
2. Hardcoding the portal name as a string almost identical to the bot's name (somewhat error-prone, since one may forget to change it)
3. Not recording the Portal's name

What do you prefer?

Reply:
I feel like extracting it from the bot's name feels a bit hack-ish, as the bot's name is how scrapy knows where to look for it, and I feel it shouldn't be inside scraping logic.

I like approach #1, but I would extract it from the URL (not that you really should, I'm being somewhat arbitrary here).

Reply:
Updated. Also improved scraping for both bots. Please, review